packages/base/source/server/events.js

```diff
@@ -5,7 +5,7 @@ Space.messaging.define(Space.messaging.Event, 'Todos', {
   },
 
   TodoCreated: {
-    id: Guid,
+    todoId: Guid,
```

Comment:
I like `target` and `source` as not only is it pretty standard for CQRS, but it communicates intent everywhere we use them. Now I think more about it, keeping away from `id` makes a lot of sense in all event props as it could be misunderstood as the id of the event!